### ISSUE TITLE
white bar at top of screen

### ISSUE THREAD

**Issue by peepo** [dnschneid/crouton#3490]
This bar appeared in the last few days, and maybe accessibility related.
is it possible/credible that it may be crouton related?

Guest mode did not fix it, there is a black 'accessibility' bar at top of screen, and the X close button does not work for me.

The only three extensions I installed are those recommended by EFF to stop online tracking
of course there are a few others that auto installed, anyway I disabled them all, and restarted, to no effect.

Does anyone know if Reset interferes with crouton?

chromebook is updated ~daily

sudo edit-chroot -all
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20170901092920~master:0216f9d1
release: xenial
architecture: amd64
xmethod: xorg
targets: kde
host: version 10097.0.0 (Official Build) canary-channel squawks 
kernel: Linux localhost 4.4.92-12034-g44d05a3e1605 #1 SMP PREEMPT Sat Nov 4 04:09:45 PDT 2017 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
chronos@localhost / $
